src/main/java/io/github/homchom/recode/mod/commands/impl/item/ImportFileCommand.java

```diff
@@ -58,8 +58,8 @@ public void register(Minecraft mc, CommandDispatcher<FabricClientCommandSource>
 
                                 while (sc.hasNextLine()) {
                                     String line = sc.nextLine();
-                                    if (line.length() > 2000) {
-                                        ChatUtil.sendMessage("Line " + (lines.size() + 1) + " is too long! (" + line.length() + " > 2000)", ChatType.FAIL);
+                                    if (line.length() > 10_000) {
```

Comment:
Is there a reason this should be raised to 10,000 specifically? How did you come up with this?

Reply:
That's the new string limit on DiamondFire